patches/server/0925-Elder-Guardian-appearance-API.patch

```diff
@@ -0,0 +1,24 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SoSeDiK <mrsosedik@gmail.com>
+Date: Tue, 11 Oct 2022 20:38:47 +0300
+Subject: [PATCH] Elder Guardian appearance API
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index 968aa80b57a31d89852c6f4bc0ec5ed4a98c6530..4712a034ea43a6dada15d68e8657c48519b3eac0 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -2932,6 +2932,13 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
+     }
+     // Paper end
+ 
++    // Paper start
++    @Override
++    public void showElderGuardian(boolean silent) {
++        getHandle().connection.send(new ClientboundGameEventPacket(ClientboundGameEventPacket.GUARDIAN_ELDER_EFFECT, silent ? 0F : 1F));
```

Comment:
Generally these "packet" api also check if the connection isn't null to avoid a potential NPE call from an unsafe place

Reply:
> Generally these "packet" api also check if the connection isn't null to avoid a potential NPE call from an unsafe place

I can do that if needed. IDE swears at me that it can't be null, and many methods (included Paper's) do not have it.

Reply:
Yep, you've right some paper API doesn't use that but not all (see sendMultiBlockChange for example)
To test that see if this throw an NPE inside the PlayerSpawnLocationEvent ?